src/style/adwaitastyle.cpp

```diff
@@ -6751,7 +6753,9 @@ bool Style::drawSliderComplexControl(const QStyleOptionComplex *option, QPainter
             StyleOptions styleOptions(palette, _variant);
 
             // colors
-            QColor base(Colors::separatorColor(styleOptions));
+            // QColor base(Colors::separatorColor(styleOptions));
+            // Fix black slider ticks for dark variant
+            QColor base(Colors::sliderTicksColor(styleOptions));
```

Comment:
Maybe just use:
`QColor base(Colors::transparentize(palette.text(), 0.5))`

Without introducing a new public method. Also, this is an unrelated change to spinboxes so maybe put it into a separate review.

Reply:
I don't think I'll have the time in the foreseeable future. I has a play over xmas, it's a far as I got, now back to work

For me it doesn't matter because it a common bug in a lot of qt dark themes with slider ticks (kvantum gets it right though)  So I implemented my own derived slider widget, override the paint event so it works with all themes now, so no more problems.

As for the changes I run it on POP with the above changes, to me it already looks much better, so. I am happy, and took the time to report it ... BUT ...
Since you know the code bas (I don't). I personally do think the slider and spinboxes should be fixed in whatever way you see fit.

I leave it to you. I raised the bug, no doubt for you it will be trivial to fix it in whatever way you see fit.

Or 

you can wait, whenever I have the incentive and/or time to dig in, could be .. whenever